### ISSUE TITLE
Feature request - support for VS code tasks

### ISSUE THREAD

**Issue by cmeyertons** [madskristensen/CommandTaskRunner#32]
Not sure exactly where to put this, but it would be great to add capability to this extension to run VS code tasks from a tasks.json

**Comment by redwyre** [comment]
yeah, this would be great since vscode has a standard format, and it would work for both (and other IDEs too)
